web/config/sync/filter.format.dbcdk_community_profiles__description.yml

```diff
@@ -0,0 +1,23 @@
+uuid: bda9e83c-d51b-40f6-a306-6cd8b0496666
+langcode: en
+status: true
+dependencies: {  }
+name: 'DBCDK Community Profiles - Description'
+format: dbcdk_community_profiles__description
```

Comment:
I wasn't sure about the different needs in each field, so this was just a more cautious decision - I would prefer to make one that is generic enough to be used in all fields that has markup but I think we would have to discuss this with the Biblo.dk team to be sure of any generalisations